### ISSUE TITLE
Featured tags location on web app

### ISSUE THREAD

**Issue by tenebrios** [mastodon/mastodon-android#798]
Where are we to find the featured hashtag list on our profiles while using the "web app".

I created a hashtag list on the web app and it's no where to be found on my profile. 

How can myself and other's locate these featured hashtag lists on our profiles?

**Comment by tenebrios** [comment]
I would imagine the featured hashtag list should appear on the 3 dot menu but as you see, it's not here.

Also, please just make this unlimited.
A limit of 10 just isn't enough


![Screenshot_20240313_173414_Chrome](https://github.com/mastodon/mastodon-android/assets/161815189/beabe651-a8f7-461a-984a-f0dd44b25e62)

**Comment by andypiper** [comment]
This is a known detail that is not visible in the Android or iOS apps. If users are interested to find featured hashtags, they can visit the profile via the web. It may be added to the mobile apps in future, but it is not a current priority.

**Comment by tenebrios** [comment]
> This is a known detail that is not visible in the Android or iOS apps. If users are interested to find featured hashtags, they can visit the profile via the web. It may be added to the mobile apps in future, but it is not a current priority.

It's actually the other way around.
The featured tag list is viewable on the Android app.
The list has to be MADE using the web app as it can't be made through the android app.
So I made my featured tag list in the web app. It appears on the android app. The issue is on the android app, initially it shows 5 out of the 10 tags on the list.
Those initial 5 tags function fine. They can be opened and there are no issues with those first 5 tags.
The problem is when you hit 'view all" to see the FULL list of 10 tags. After you hit view all and THEN try to open any one of the 10 tags, the app closes out.

Also note that the featured tag list is nowhere to be found on the web app version.

**Comment by andypiper** [comment]
Apologies for my misunderstanding.
Noted that there may be an issue with more than 5 tags.

If this is prioritised we will come around to look further at this, but so far it does not seem to be a major concern.

Web app issues can be raised in the main repository, but I believe that this is covered in https://github.com/mastodon/mastodon/issues/21270 so does not need an additional report there.

**Comment by tenebrios** [comment]
> Apologies for my misunderstanding. Noted that there may be an issue with more than 5 tags.
> 
> If this is prioritised we will come around to look further at this, but so far it does not seem to be a major concern.
> 
> Web app issues can be raised in the main repository, but I believe that this is covered in [mastodon/mastodon#21270](https://github.com/mastodon/mastodon/issues/21270) so does not need an additional report there.

A couple weeks ago the featured tag list was actually not working properly.
The way it's supposed to work is we create a list of tags and when we open any one of those tags on our page, it only shows us what we posted using that tag. It wasn't doing that. It was showing the posts that everyone on the app made with those tags.
I mentioned it as an issue and someone corrected that issue fairly quickly.
Now when I open the 5 tags that are able to be opened (before pressing view all), they only show the posts I made with that particular tag.

I mention this because the turn around on that fix was quick.
I just wonder why fixing this bug can't happen as quickly.
I thought that developers usually want to address bugs quickly. Especially ones that close out the app
